Team9Project/src/com/indragie/cmput301as1/ApprovalTabFragment.java

```diff
@@ -0,0 +1,16 @@
+package com.indragie.cmput301as1;
+
+import java.util.List;
+ 
+ 
+/** This is a listfragment class */
+public class ApprovalTabFragment extends BaseTabFragment {
+	
+    
+	@SuppressWarnings("unchecked")
+	@Override
+	public void load(){
+		listModel = new ListModel<ExpenseClaim>(EXPENSE_APPROVAL_FILENAME, getActivity());
+		listModel.addObserver((TypedObserver<CollectionMutation<ExpenseClaim>>) getActivity());
```

Comment:
Repeat from comment below:

The observer should be the fragment (i.e. `this`), not the activity, since you moved all of the code from the activity into the fragment, including the observer callbacks.